tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,45 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource    ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+
+*** Test Cases ***
+Any OpenShift User Should Also Be A RHODS User
+    [Documentation]    Any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       ODS-1164
+    Set Library Search Order    SeleniumLibrary
+    Open Browser    ${OCP_CONSOLE_URL}    ${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To Openshift    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    ${OCP_ADMIN_USER.AUTH_TYPE}
+    Go To Config-Maps
+    Select Project By Name    redhat-ods-applications
+    Select Rhods-Groups-Config
+    Verify Groups ConfigMap Contains The Expected OCP Groups
+    Spawn Notebook
+
+
+*** Keywords ***
+Go To Config-Maps
+    [Documentation]    Go to Workloads and Select Configmaps
+    Click Button    xpath=//button[contains(.,'Workloads')]
+    Click Link    xpath=//a[contains(.,'ConfigMaps')]
+
+Select Rhods-Groups-Config
+    [Documentation]    Select rhods-groups-config and click
+    Wait Until Page Contains Element    xpath=//input[@class='pf-c-form-control co-text-filter']
+    Input Text    xpath=//input[@class='pf-c-form-control co-text-filter']    rhods-groups-config
+    Wait Until Page Contains Element    xpath=//a[contains(text(),'rhods-groups-config')]
+    Click Link    xpath=//a[contains(text(),'rhods-groups-config')]
+
+Verify Groups ConfigMap Contains The Expected OCP Groups
+    [Documentation]    Verify that ConfigMap contains expected Groups
+    Wait Until Page Contains Element    xpath=//*[@class='co-pre-wrap co-copy-to-clipboard__text']
+    Element Text Should Be    xpath=//*[@id="content-scrollable"]//dd[1]/div/pre    cluster-admin
+    Element Text Should Be    xpath=//*[@id="content-scrollable"]//dd[2]/div/pre    system:authenticated
```

Comment:
these checks will fail on RHODS v1.7.0. Maybe it would be necessary to use an IF/ELSE statement to perform checks based on the RHODS versions